src/Tribe/Utils/Post_Thumbnail.php

```diff
@@ -147,6 +147,8 @@ public function get_image_sizes() {
 	 * @return array An array of objects containing the post thumbnail data.
 	 */
 	public function fetch_data() {
+		static $cache_thumbnail = [];
```

Comment:
:no_entry_sign: Short array syntax is not allowed

<sub>hash: 1f22ff14625304fc4856082dfcb6492f</sub>